PotentialGapPlanner:
    min_resoln: 1
    gap_diff: 0.1
    global_plan_lookup_increment: 30
    goal_tolerance: 0.5
    global_plan_change_tolerance: 0.1
    follow_the_gap_visparsing: True
    close_gap_visparsing: True

    epsilon2: 0.20
    epsilon1: 0.20
    r_inscr:  0.20

    niGen_s: True
    config_sanity_val: 100
    ctrl_ahead_pose: 2 # 3 for Gazebo 
    reduction_threshold: 1.5708
    max_idx_diff: 128
    feasi_inflated: True
    v_lin_x_const: 0.0
    v_lin_y_const: 0.0
    k_drive_x: 2
    k_drive_y: 3.5
    projection_inflated: False
    planning_inflated: False
    fig_gen: False
    k_po: 1
    k_po_turn: 1
    inf_ratio: 1.1

    r_min: 0.36
    r_norm: 1
    r_norm_offset: 0

    synthesized_frame: True
    scale: 1
    integrate_maxt: 10
    integrate_stept: 0.01

    num_feasi_check: 30
    projection_operator: True
    line: False

    # man_ctrl: True
    # man_x: 0.2